spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -154,12 +154,16 @@ definitions:
             fill: u8
             size: 4
             desc: The last 4 bytes of the certificate's SHA-1 fingerprint
+        - n_signature_bytes:
```

Comment:
This is changing the structure of an existing message by inserting a new field part way through rather than at the end. Shouldn't this require a bump to the message ID? Deprecate the old one to maintain backwards compatibility?

Reply:
@Blast545 and I are the only consumers of the message at the moment so it should be okay

Reply:
Yeah, we had a meeting to talk about this and it shouldn't be any issues changing this.